api/base/exceptions.py

```diff
@@ -1,5 +1,35 @@
-from rest_framework.exceptions import APIException
+
 from rest_framework import status
+from rest_framework.exceptions import APIException
+
+
+def json_api_exception_handler(exc, context):
+    """ Custom exception handler that returns errors object as an array """
+
+    from rest_framework.views import exception_handler
```

Comment:
If the exception_handler is imported outside of the custom exception handler, tests return `django.core.exceptions.ImproperlyConfigured: Requested setting CACHES, but settings are not configured. You must either define the environment variable DJANGO_SETTINGS_MODULE or call settings.configure() before accessing settings.`